README.md

```diff
@@ -184,197 +184,202 @@ mdsf init
-| ufmt                 | [https://github.com/omnilib/ufmt](https://github.com/omnilib/ufmt)                                                           |
-| uiua_fmt             | [https://github.com/uiua-lang/uiua](https://github.com/uiua-lang/uiua)                                                       |
-| usort                | [https://github.com/facebook/usort](https://github.com/facebook/usort)                                                       |
-| veryl_fmt            | [https://github.com/veryl-lang/veryl](https://github.com/veryl-lang/veryl)                                                   |
-| vhdl-style-guide     | [https://github.com/jeremiah-c-leary/vhdl-style-guide](https://github.com/jeremiah-c-leary/vhdl-style-guide)                 |
-| vlang_fmt            | [https://vlang.io](https://vlang.io)                                                                                         |
-| xmlformat            | [https://github.com/pamoller/xmlformatter](https://github.com/pamoller/xmlformatter)                                         |
-| xmllint              | [http://xmlsoft.org/xmllint.html](http://xmlsoft.org/xmllint.html)                                                           |
-| xo                   | [http://github.com/xojs/xo](http://github.com/xojs/xo)                                                                       |
-| yamlfix              | [https://github.com/lyz-code/yamlfix](https://github.com/lyz-code/yamlfix)                                                   |
-| yamlfmt              | [https://github.com/google/yamlfmt](https://github.com/google/yamlfmt)                                                       |
-| yapf                 | [https://github.com/google/yapf](https://github.com/google/yapf)                                                             |
-| yew-fmt              | [https://github.com/its-the-shrimp/yew-fmt](https://github.com/its-the-shrimp/yew-fmt)                                       |
-| zigfmt               | [https://ziglang.org/](https://ziglang.org/)                                                                                 |
-| ziggy_fmt            | [https://ziggy-lang.io/documentation/ziggy-fmt/](https://ziggy-lang.io/documentation/ziggy-fmt/)                             |
-| zprint               | [https://github.com/kkinnear/zprint](https://github.com/kkinnear/zprint)                                                     |
+`mdsf` currently supports 192 commands. Feel free to open an issue/pull-request if your favorite tool is missing! 😃
+
+| Name                   | Command                                                                                |
+| ---------------------- | -------------------------------------------------------------------------------------- |
+| `alejandra`            | `alejandra --quiet PATH`                                                               |
```

Comment:
You removed the formater link, maybe you could bring it back

```suggestion
| [`alejandra`](https://github.com/kamadorueda/alejandra)            | `alejandra --quiet PATH`                                                               |
```

Reply:
I am gonna add both fields. It just requires a rewrite of how I generate the table, which I haven't gotten to yet 😀

Reply:
Great